src/machines/modelingMachine.ts

```diff
@@ -1187,8 +1188,11 @@ export const modelingMachine = createMachine(
         )
         if (err(varDecNode)) return
         const sketchVar = varDecNode.node.declarations[0].id.name
-        const sketchGroup = kclManager.programMemory.get(sketchVar)
-        if (sketchGroup?.type !== 'SketchGroup') return
+        const sketchGroup = sketchGroupFromKclValue(
+          kclManager.programMemory.get(sketchVar),
+          sketchVar
+        )
+        if (trap(sketchGroup)) return
```

Comment:
I think `trap` will throw up a toast, just a heads up in case that isn't the intended behavior, since I saw you use `err` elsewhere.

Reply:
The problem is that switching to `err` would silently swallow the error. Do we have another function that prints to the console but doesn't show a toast?

Reply:
Seems like we can use `trap` with suppress = true for that.